firmware/src/widgets/stockwidget/StockWidget.cpp

```diff
@@ -107,7 +108,7 @@ void StockWidget::buttonPressed(uint8_t buttonId, ButtonState state) {
 }
 
 void StockWidget::displayStock(int8_t displayIndex, StockDataModel &stock, uint32_t backgroundColor, uint32_t textColor) {
-    Serial.println("displayStock - " + stock.getSymbol() + " ~ " + stock.getCurrentPrice());
+    Log.infoln("displayStock - %s ~ %d", stock.getSymbol(), stock.getCurrentPrice());
```

Comment:
`stock.getCurrentPrice()` returns a float, not a decimal - so you probably need a `%9.6f` type format or pass a "digits" so you get a string